src/io/flutter/view/InspectorPanel.java

```diff
@@ -539,6 +547,91 @@ public void showProperties(DiagnosticsNode diagnostic) {
     }
   }
 
+  private static SimpleTextAttributes textAttributesForLevel(DiagnosticLevel level) {
+    switch (level) {
+      case hidden:
+      case fine:
+        return SimpleTextAttributes.GRAYED_ATTRIBUTES;
+      case warning:
+        // TODO(jacobr): would be nice to use a yellow color for level warning.
+      case error:
+        return SimpleTextAttributes.ERROR_ATTRIBUTES;
+      case debug:
+      case info:
+      default:
+        return SimpleTextAttributes.REGULAR_ATTRIBUTES;
+    }
+  }
+
+  private static class PropertyNameRenderer extends ColoredTableCellRenderer {
+    @Override
+    protected void customizeCellRenderer(JTable table, @Nullable Object value, boolean selected, boolean hasFocus, int row, int column) {
+      if (value == null) return;
```

Comment:
Might it be safer to replace this with:

    if (!(value instance of DiagnosticNode)) return;

?

Reply:
I'd rather throw if we get the wrong type of value rather than quietly filtering them out.